fetch-todolist/src/component/Filters.jsx

```diff
@@ -0,0 +1,25 @@
+import {memo} from 'react';
+const FiltersComponent = ({handleStatusFilter,searchTask,handleOrder}) => {
+    
+    
+    
+    return ( <>
```

Comment:
create common select component

Reply:
created a component named SelectBox and using it by passing arrays to it, in the latest commit.